src/main/common/sensor_alignment_impl.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+//
+// alignment bitmasks
+//
+
+#define ALIGNMENT_ROTATION_WIDTH 2
```

Comment:
What is the purpose of this file?

Reply:
to extract the non-API code from the API code in a way that can be unit tested.

Reply:
This is a very weird concept of what a unit test should be testing that you have - I cannot see any place where any of this is actually exposed in the API of a unit, so not sure why we would want to have to spend time on maintaining 'unit tests' that verify something that is not the exposed behaviour of a unit.
Maybe refactor this to make it a unit of its own if you think it should be, or else ditch the non-unit tests?